pythonforandroid/recipes/libxml2/__init__.py

```diff
@@ -5,27 +5,18 @@
 from os.path import join
 
 
-class Libxml2Recipe(CythonRecipe):
-    version = '2.9.4'
+class Libxml2Recipe(Recipe):
+    version = '2.9.3'
     url = 'ftp://xmlsoft.org/libxml2/libxml2-{version}.tar.gz'
     name = 'libxml2'
     depends = [('python2', 'python3crystax')]
 
     def get_recipe_env(self, arch):
         env = super(Libxml2Recipe, self).get_recipe_env(arch)
 
-        try:
-            sh.sed('runtest\$(EXEEXT) \\/ \\/', 'Makefile')
-        except:
-            pass
-        try:
-            sh.sed('testrecurse\$(EXEEXT)$//', 'Makefile')
-        except:
-            pass
-        try:
-            sh.make('-j', '$MAKE_JOBS')
-        except:
-            pass
+        sh.sed('runtest$(EXEEXT) \/ \/', 'Makefile')
+        sh.sed('testrecurse$(EXEEXT)$//', 'Makefile')
+        sh.make('-j', '5')
```

Comment:
These lines should be placed in the `build_arch` method, and you need to navigate to the right directory, something like

    def build_arch(self, arch):
        with current_directory(self.get_build_dir(arch)):
            sh.sed(....)